dot/types/grandpa_test.go

```diff
@@ -7,55 +7,129 @@ import (
 	"testing"
 
 	"github.com/ChainSafe/gossamer/lib/common"
+	"github.com/ChainSafe/gossamer/lib/crypto/ed25519"
 	"github.com/ChainSafe/gossamer/pkg/scale"
 
 	"github.com/stretchr/testify/require"
 )
 
-func TestEncodeGrandpaVote(t *testing.T) {
-	exp := common.MustHexToBytes("0x0a0b0c0d00000000000000000000000000000000000000000000000000000000e7030000")
-	var testVote = GrandpaVote{
+func mustHexTo64BArray(t *testing.T, inputHex string) (outputArray [64]byte) {
```

Comment:
Since this function is defined twice and seems rather general purpose, I think we can add this in `lib/common`, you can move this function there and introduce common.Signature

`type Signature [64]byte`